Description: Avoid use of npm during build
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: no
Last-Update: 2018-11-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -85,6 +85,6 @@
 	git checkout lunr.* *.json
 
 node_modules: package.json
-	${NPM} -s install
+#	${NPM} -s install
 
 .PHONY: test clean docs reset perf/*_perf.js test/inspect
